ansible_community_steering_committee.rst

```diff
@@ -0,0 +1,110 @@
+proposals (feature idea, new policy in collection world), new collection
+requests, empower contributors, maintainers & reviewers and other
+technical aspects regarding inclusion and packaging.
+
+Committee Member Responsibilities:
+==================================
+
+The members in the committee handle the below:
+
+- Design policy and procedure for the collection world.
+- Review things (collections fornow) for compliance with the policies. 
+- Help create our deliverables (community.general, community.network, the ansible-tarball, docs website).
+- Ballot on approval changes to policy and new proposals.
+- Identify and train individuals from the community who are interested in contributing to Ansible.
+
+Members:
+========
+
+1.  Felix Fontein (felixfontein)
+2.  Tadej Borovšak (tadeboro)
+3.  James Cassell (cyberpear)
+4.  John Barker (gundalow)
+5.  Andrew Klychkov (andersson007_)
+6.  Alicia Cozine (acozine)
+7.  Sorin Sbarnea (zbr)
+8.  Jill Rouleau (jillr)
+9. Brad Thornton (cidrblock)
+10. Dylan Silva (thaumos)
+
+*Key: Full name (IRC Nickname)*
+
+-  Currently, all the Steering Committee members are selected based on
+   their active contribution to the Ansible Project.
+-  In future, the selection process will be based on the election model
+   through community.
+
+New Policy/Proposals & Inclusion Requests:
+==========================================
+
+Requests regarding new collection inclusion requests should be submitted
+to this `GitHub
+discussion <https://github.com/ansible-collections/ansible-inclusion/discussions/new>`__
+and requests regarding new policy/proposals should be submitted through
+a new pull request to
+`ansible-collections/overview <https://github.com/ansible-collections/overview>`__
+GitHub repo.
+
+Also, we earlier used a `Community Working Group Meeting
+Agenda <https://github.com/ansible/community/issues/539>`__ to track
+agenda topics that needed to be discussed in the weekly Ansible
+Community IRC meetings. Now, to categorize each topic separately we’ve
+created a new GitHub repo
+“`community-topics <https://github.com/ansible-community/community-topics>`__”.
+If you want to discuss an idea, improvements or submitted new
+Policy/Proposals & New Collection Inclusion Requests then create a `new
+issue <https://github.com/ansible-community/community-topics/issues/new>`__
+in the `repo <https://github.com/ansible-community/community-topics>`__
+as a topic, and it’ll be discussed publicly in the weekly IRC meeting.
+After every meeting, the meeting minutes/summaries and meeting logs will
+be posted to the `original
+issue <https://github.com/ansible/community/issues/539>`__.
+
+If you cannot join the meeting or from another time zone then you can
+directly use GitHub Discussions feature to discuss your new
+proposals/collection inclusion requests and the steering committee will
+discuss the final conclusions during the weekly IRC meeting and then
+will vote on them.
+
```

Comment:
```suggestion
```
As we are moving to the async discussion / voting model, i think we could remove this for now (the async voting has already started happening)